.github/workflows/pull_request.yaml

```diff
@@ -77,4 +77,15 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.STAKATER_GITHUB_TOKEN }}
         SLACK_WEBHOOK_URL: ${{ secrets.STAKATER_DELIVERY_SLACK_WEBHOOK }}
-  
+  unittest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          version: '3.11.0'
+          token: ${{secrets.STAKATER_GITHUB_TOKEN}}
+      - run: helm dependency build
```

Comment:
remove this run, not needed since no dependencies exist in the helm chart

Reply:
updated